packages/scss/src/components/_nav-side.scss

```diff
@@ -278,6 +309,7 @@
 			justify-content: center;
 			padding: _theme("spacings.small") _theme("spacings.smaller");
 			position: relative;
+			text-align: center;
 
 			.navSide-item-link-title {
 				margin: .8em 0 0;
```

Comment:
```suggestion
				margin: .8rem 0 0;
```

Reply:
I will harmonize in `em` to ensure consistency between products.